README.md

```diff
@@ -1,34 +1,194 @@
-# Membrane Template Plugin
+# Membrane Telmetry Metrics
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_template_plugin.svg)](https://hex.pm/packages/membrane_template_plugin)
-[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_template_plugin)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane_template_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_template_plugin)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_telemetry_metrics.svg)](https://hex.pm/packages/membrane_telemetry_metrics)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_telemetry_metrics)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_telemetry_metrics.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_telemetry_metrics)
 
-This repository contains a template for new plugins.
-
-Check out different branches for other flavours of template.
+This repository contains a tool for generating metrics.
```

Comment:
What about

```
It wraps `:telemetry.execute` to not introduce performance overhead when telemetry is turned off and provides generic reporter responsible for creating reports as Elixir maps according to provided tags. 
```

I also think that we should describe somewhere (maybe in Membrane Guide or in membrane_core) what is our approach to metrics i.e. that we are using everywhere `membrane_telemetry_metrics` and that this is the preffered way to collect statistics from Membrane plugin. Then we should redirect to this repo to teach how to add metrics and generate reports.